packages/setup_template.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
```

Comment:
so you used this to generate all of the setup.py??

Reply:
Yes. And then I made changes where needed.